templates/page_object.rb

```diff
@@ -0,0 +1,7 @@
+class PageObject
```

Comment:
Greg, that sounds great, but I'm not quite sure what you mean or what you'd change. Perhaps some code would clear it up, or maybe you can write a little more?